### PR TITLE
[Verifiable Credentials 2019] Liaison with ISO SC17 WG10

### LINKED ISSUE #245: [Verifiable Credentials 2019] Laison with ISO SC17 WG10
Would it be possible to add to section 4.2 External Organisations the following text

ISO SC17 WG10
Ensure that the mobile driving licenses being modeled and expressed by the ISO SC17 WG10  community are compatible with the Verifiable Credentials WG.

### PR DESCRIPTION
**Pull request by iherman**
PR version of issue #245.

If merged, fix #245.

Cc @David-Chadwick

### CODE REVIEW

**Comment by iherman** [comment]
Cc: @msporny @burnburn @stonematt

**Review by David-Chadwick — CHANGES_REQUESTED** [review]
I think it needs a CR/LF inserting to reduce the characters per line

**Comment by iherman** [comment]
@David-Chadwick HTML will take care of that, see:

https://raw.githack.com/w3c/charter-drafts/VC19-ISO-SC17-liaison/verifiable-credentials-2019.html

B.t.w., does the WG have a Web site that could be linked to the text?

File: verifiable-credentials-2019.html [hunk 1]
```diff
@@ -246,6 +246,8 @@ <h3 id="external-coordination">External Organizations</h3>
              <dd>Ensure that the badges being modeled and expressed by the Open Badges community are compatible with the Verifiable Credentials WG.</dd>
            <dt><a href="https://www.credentialtransparencyinitiative.org/">Credentials Transparency Initiative</a></dt>
             <dd>Ensure that the credentials being modeled and expressed by CTI are compatible with the work of the Verifiable Credentials WG.</dd>
+           <dt>ISO SC17 WG10<dt>
+           <dd>Ensure that the mobile driving licenses being modeled and expressed by the ISO SC17 WG10 community are compatible with the work of the Verifiable Credentials WG</dd>
```
Comment (msporny) [inline verifiable-credentials-2019.html]:
```suggestion
           <dd>Ensure that the mobile driving licenses being modeled and expressed by the ISO SC17 WG10 community are compatible with the work of the Verifiable Credentials WG.</dd>
```

**Review by msporny — CHANGES_REQUESTED** [review]
Minor punctuation change, otherwise, LGTM.

**Comment by David-Chadwick** [comment]
On 01/12/2019 10:55, Ivan Herman wrote:
>
> @David-Chadwick <https://github.com/David-Chadwick> HTML will take 
> care of that, see:
>
> https://raw.githack.com/w3c/charter-drafts/VC19-ISO-SC17-liaison/verifiable-credentials-2019.html
>
> B.t.w., does the WG have a Web site that could be linked to the text?
>
Not that I am aware of. It has pages for its deliverables and meeting 
minutes etc. Also I have created a Wikipedia page here 
[https://en.wikipedia.org/wiki/Verifiable_credentials].

Manu can give you the definitive answer about a WG web site.

Kind regards

David

**Comment by msporny** [comment]
@David-Chadwick wrote:
> Manu can give you the definitive answer about a WG web site.

@iherman is asking about the website for the ISO-SC17 group... which, I think, is technically  ISO/IEC JTC 1/SC 17/WG 10... which is listed here: https://www.iso.org/committee/45144.html

File: verifiable-credentials-2019.html [hunk 2]
```diff
@@ -246,6 +246,8 @@ <h3 id="external-coordination">External Organizations</h3>
              <dd>Ensure that the badges being modeled and expressed by the Open Badges community are compatible with the Verifiable Credentials WG.</dd>
            <dt><a href="https://www.credentialtransparencyinitiative.org/">Credentials Transparency Initiative</a></dt>
             <dd>Ensure that the credentials being modeled and expressed by CTI are compatible with the work of the Verifiable Credentials WG.</dd>
+           <dt>ISO SC17 WG10<dt>
```
Comment (msporny) [inline verifiable-credentials-2019.html]:
```suggestion
           <dt><a href="https://www.iso.org/committee/45144.html">ISO/IEC JTC 1/SC 17/WG 10</a><dt>
```

**Comment by iherman** [comment]
Yes indeed! Thanks Manu, I will update the file on the repo tomorrow.

Ivan

---
Ivan Herman
World Wide Web Consortium
http://www.w3.org/People/Ivan/
ORCID: 0000-0003-0782-2704

**Review by msporny — CHANGES_REQUESTED** [review]
Add link to ISO SC17 WG10.

**Review by plehegar — APPROVED** [review]

**Comment by iherman** [comment]
Thanks @msporny I have committed both of your suggestions.

**Review by plehegar — APPROVED** [review]

**Pull request merged**